contracts/masset/MassetStructs.sol

```diff
@@ -73,6 +80,18 @@ struct FeederData {
     WeightLimits weightLimits;
 }
 
+struct MassetData {
+    uint256 swapFee;
+    uint256 redemptionFee;
```

Comment:
No `govFee` like the Feeder Pools?

Reply:
`govFee` is essentially equivalent to the `savingsRate` in SavingsManager, which applies to mAsset revenue